app/models/booking.rb

```diff
@@ -0,0 +1,10 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :pet
+
+  validates :start_date, presence: true
+  validates :end_date, presence: true
```

Comment:
this should be validated to only dates after start_date

Reply:
also add Pandit authorization for bookings